crates/pyluwen/python_unit_tests/wh_ubb_reset_test.py

```diff
@@ -0,0 +1,30 @@
+import time
+from pyluwen import detect_chips, run_wh_ubb_ipmi_reset, run_ubb_wait_for_driver_load
+
+def main():
+    # Detect chips
+    devices = detect_chips()
+    for i, dev in enumerate(devices):
+        if dev.as_wh():
+            # arc_msg to set it into the A3 state
+            dev.as_wh().arc_msg(0xA3, wait_for_done=False)
+            print("arc_msg sent to WH chip")
```

Comment:
I think this needs to be in the final tool too, not just in the unit test?

That being said, also:

Since you're printing for every chip, maybe print the index you're sending it to?

Nit / future change: Should this be moved into the ipmi_reset function, as a step before sending the ipmi message?

Of course, later stage would be to reduce it down to only do the ARC message for the chips being reset -- currently it's all or nothing. When that happens, it'd make sense to have it there instead.

Reply:
Per internal follow-up, my confusion on unit test vs real call to the ARC A3 messages is due to that being added to tt-smi instead of here (it's mostly temporary code since future FW won't need it).

Let's get this merged and get going, even if it can be refactored in a few ways later on.